biggraphite/test_utils_elasticsearch.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# Copyright 2018 Criteo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Elasticsearch utilities factoring code across tests.
+
+The dependencies for this module are not included in requirements.txt or in the package
+dependencies, instead one needs the elements of tests-requirements.txt .
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import sys
+import logging
+
+
+HAS_ES_HOME = bool(os.getenv("ES_HOME"))
+ES_HOSTPORT = os.getenv("ES_HOSTPORT")
+HAS_ELASTICSEARCH = HAS_ES_HOME or ES_HOSTPORT
+
+# Only try to import cassandra if we are going to use it. This is better
+# than using try/except because the failure case is easier to handle.
+if HAS_ES_HOME:
+    from testing import elasticsearch as testing_elasticsearch
+
+
+class ElasticsearchHelper():
+    """Helper for an ephemeral Elasticsearch cluster."""
+
+    INDEX = "testindex"
+
+    @classmethod
+    def get_accessor_settings(cls):
+        """Prepare accessor settings for Cassandra driver."""
+        return {
+            "elasticsearch_index": cls.INDEX,
+            "elasticsearch_hosts": cls.hosts,
+            "elasticsearch_port": cls.port,
+            "elasticsearch_timeout": 60
+        }
+
+    @classmethod
+    def setUpClass(cls):
+        """Create the test Elasticsearch Cluster as cls.elasticsearch."""
+        cls.elasticsearch = None
+        if ES_HOSTPORT:
+            host, cls.port = ES_HOSTPORT.split(':')
+            cls.hosts = [host]
```

Comment:
For consistency with other variables (eg: #367), what about using full name (`ELASTICSEARCH`) instead of `ES`?